### PR TITLE
[Term Entry] Neural Networks ai Learning Rate Scheduling

### LINKED ISSUE #4530: [Term Entry] Neural Networks ai Learning Rate Scheduling
### Reason/inspiration (optional)

"We would like a new term entry in the `AI` concept for [neural-network](https://www.codecademy.com/resources/docs/ai/neural-networks): Learning Rate Schedule. The entry should go in a new file under `docs/content/ai/concepts/neural-networks/terms/learning-rate-schedule/learning-rate-schedule.md`.

The entry should describe the term and how it is used. Also include: 
- An "Example" section that explains to readers how to use concept for stochastic gradient descent.
- Should contain information for this concept and syntax.
- Should have a Python codebyte with a working code snippet showcasing this concept.
Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue."

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by mamtawardhani**
### Description

Added a new Term Entry for Neural Networks ai Learning Rate Scheduling

### Issue Solved

Closes #4530 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by mamtawardhani** [comment]
> Only a few comments for u to work on Mamta :v

Fixed them 😅
